### PR TITLE
SW-3507 Add SelectPhotos component to web-components

### PR DESCRIPTION
**Pull request by constanzauanini**
Screenshot 2023-04-28 at 12 13 17

### CODE REVIEW

**Review by karthikbtf — COMMENTED** [review]
Export this from index.ts
Also,, thoughs on renaming this to PhotoChooser (like FileChooser).. I called it SelectPhotos initially without much thought.
Maybe we could at some point make this a FileChooser and pass the accepted types as props.. (right now we can't since it has specific photo related iconography in the drag-n-drop pad.. we'd have to customize those too). Just thinking aloud.

**Comment by constanzauanini** [comment]
> ndex.ts

Yes, I think we can adapt this to accept any file in the future if we need it

**Review by karthikbtf — APPROVED** [review]

File: src/components/PhotoChooser.tsx [hunk 1]
```diff
@@ -0,0 +1,229 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { Box, Typography, useTheme } from '@mui/material';
+import { Theme } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+import { useDeviceInfo } from '../utils';
+import ErrorBox from './ErrorBox/ErrorBox';
+import Button from './Button/Button';
+import Icon from './Icon/Icon';
```
Comment (karthikbtf) [inline src/components/PhotoChooser.tsx:8]: @constanzauanini I don't know how I missed this or how this even builds.. 
let's redo this please

Comment (karthikbtf) [reply src/components/PhotoChooser.tsx:8]: oops,, wrong place comment

Comment (karthikbtf) [reply src/components/PhotoChooser.tsx:8]: please ignore

**Pull request merged**